File: Unit9/Unit9_CodeFiles/Unit9.R
# Set File Path
dataFile <- file.path("C:\\Users\\laure\\OneDrive\\Documents\\Purdue",
                      "\\IN300 - Programming for Data Analysis\\",
                      "Unit9\\IN300_Dataset1.csv")

# Read file
data <- read.csv(dataFile)

# Sort the data
printSortedOutput <- function(temp_T) {
  tempSort <- rev(sort(table(temp_T)))
  return(tempSort)
}

# Print Source IP counts
print("Source IP")
print(printSortedOutput(data$Source))

# Print Destination IP counts
print("Destination IP")
print(printSortedOutput(data$Destination))

# Print Protocol counts
print("Protocols")
print(printSortedOutput(data$Protocol))

File: Unit9/Unit9_CodeFiles/Unit9.py
import pandas as pd

# Prints message and value in a formatted method
def printResults(message, value):
    newline = "\n"
    print(newline, newline, message, newline, value)

# Set File Path
dataFile = "C:\\Users\\laure\\OneDrive\\Documents\\Purdue" \
            "\\IN300 - Programming for Data Analysis\\Unit9" \
           "\\IN300_Dataset1.csv"

# Read file into a data frame
data = pd.read_csv(dataFile, sep=',', decimal='.', header=0)

# Print the source IP frequencies
printResults("Source IP", data['Source'].value_counts())

# Print the destination IP frequencies
printResults("Destination IP", data['Destination'].value_counts())

# Print the protocol frequencies
printResults("Protocol", data['Protocol'].value_counts())